framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -2397,32 +2406,88 @@ public boolean isSupportedQualifier(/*@Nullable*/ AnnotationMirror a) {
                 this.getQualifierHierarchy().getTypeQualifiers(), a);
     }
 
-    /** Add the annotation {@code clazz} as an alias for the annotation {@code type}. */
-    protected void addAliasedAnnotation(Class<?> alias, AnnotationMirror type) {
-        addAliasedAnnotation(alias.getCanonicalName(), type);
+    /** Adds the annotation {@code aliasClass} as an alias for the annotation {@code type}. */
+    protected void addAliasedAnnotation(Class<?> aliasClass, AnnotationMirror type) {
+        addAliasedAnnotation(aliasClass.getCanonicalName(), type);
+    }
+
+    /**
+     * Adds the annotation whose canonical name is given by {@code aliasName} as an alias for the
```

Comment:
Change the `canonical name` to `fully-qualified name`? Isn't it the fully-qualified class name of the alias that is given as the first argument?